### DISCUSSION TITLE
Quick Links and other questions?

### DISCUSSION THREAD

**Discussion by Rio517** [tfwright/live_admin#73] [Q&A] [answered]
Hi, 

I was checking out 0.11 - lots of improvements.  Thanks!

I had a few questions:

* Is there a way to specify and order the columns rendered in the index table independently of those in the show/edit page?
* I noticed that this version replaces the three dots in the first column with checkboxes. For our use case, we don't use multi-select. Is there a way to remove that column?
* We typically jumped into editing records from the list page and previously click on this epsilon, then click edit to get to that page. In 0.11.0, it is rather tedius to click to the show page, scroll down, then click on edit. Is there a way to restore that functionality or to otherwise add an arbitrary column I could inject a set of buttons that linked to other pages based on the current record's id, including pages like the live_admin edit page?
* Finally, is there a way to use search for equals rather than contains?  For example, if I had books with author_id, I'd love to search for author_id of exactly "4" instead of getting results for 4, 44, 14, 41, etc.  `~p"/admin/books?s=author_id:#{record.author_id}`

Thanks!

**Comment by tfwright** [comment, marked as answer]
> Is there a way to specify and order the columns rendered in the index table independently of those in the show/edit page?

No, the order is determined by the Ecto schema definition for both. You would need to override one of the components to control the order, unless you can come up with a clever way to use CSS overrides

> Is there a way to restore that functionality or to otherwise add an arbitrary column I could inject a set of buttons that linked to other pages based on the current record's id

No and no, but you could use the `render_with` config to make one of the fields include a link.

> is there a way to use search for equals rather than contains?

No, although an "advanced" search is a feature I would be open to adding

**Reply by tfwright** [reply]
I realized I could easily support custom like matching so if you want to try it out it's in main. Now if you search for something like `blah%` it will only return matching starting with `blah`. See https://www.w3schools.com/sql/sql_like.asp

**Reply by Rio517** [reply]
Coming back to this...  I'm surprised there isn't a way to modify the Ecto schema before it is passed to be rendered into a table. Too bad.

In terms of using `render_with` to make one field include a link. I am already doing that on four columns. In my project, it is starting to feel unsustainable as we add new features. I'm planning on using "render_with" overriding one column to include a dropdown with more links. I really think users could benefit from a built-in capability. 

Thanks for the fast responses, and sorry for my slow responses!

Wishing you the best!
